### PR TITLE
Serialize entities

### PR DESCRIPTION
**Pull request by sriram1996**
Auto-Save states

### CODE REVIEW

File: src/app/util/uuid.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+/**
+ * 
+ * 
+ * @param {any} a 
+ * @return {string}
+ */
+function uuid(a) {
+    return a?(a^Math.random()*16>>a/4).toString(16):([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, b);
```
Comment (brianbad) [inline src/app/util/uuid.js]: What the actual fuck u doin bruh?

**Comment by anand-bala** [comment]
@brianbad  why did you close without merging? This is legit code.

**Comment by brianbad** [comment]
Oh I apologize. I thought this was just from our meeting where we tested the annotations feature.

**Comment by anand-bala** [comment]
:neutral_face:

**Review by brianbad — COMMENTED** [review]
Looks good for now. Remember you'll have to add player-score, time-survived, etc sometime later.

**Review by brianbad — APPROVED** [review]

**Pull request merged**